darwin/future/data_objects/properties.py

```diff
@@ -147,7 +147,7 @@ class SelectedProperty(DefaultDarwin):
         value (str): Value of the property
     """
 
-    frame_index: int
+    frame_index: Optional[int] = None
```

Comment:
This is the only meaningful change in this PR, everything else is black or ruff formatting / linting